Tests/Specs/change-streams/tests/unified/change-streams.json

```diff
@@ -1,6 +1,15 @@
 {
   "description": "change-streams",
-  "schemaVersion": "1.0",
+  "schemaVersion": "1.4",
```

Comment:
this was a manual change to prevent these from being run due to the "unified" addition to the test filter provided in run-serverless-tests.sh.
the latest version of the file has the same runOnRequirements to skip the tests on the serverless, but I can't sync the whole thing without making a number of other changes to skip new tests and support schema version 1.7, so I just copied this part in for now.